scene/gui/rich_text_label.cpp

```diff
@@ -3250,6 +3254,9 @@ void RichTextLabel::add_image(const Ref<Texture2D> &p_image, int p_width, int p_
 	item->key = p_key;
 	item->tooltip = p_tooltip;
 
+	item->owner = this;
+	item->image->connect_changed(callable_mp(this, &RichTextLabel::_texture_changed), CONNECT_REFERENCE_COUNTED);
```

Comment:
Why it's not connected to `queue_redraw()` directly?

Reply:
This would make sense with the original version, but now it's doing more stuff.